scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -12,44 +12,46 @@
 
 {% block body %}
 <nav class="navbar sticky-top navbar-expand-lg navbar-light bg-light">
-  <!-- Brand and toggle get grouped for better mobile display -->
-  <a class="nav-link" href="{{ url_for('cases.case', institute_id=institute._id, case_name=case.display_name) }}" target="_blank">Case {{case.display_name}}</a>
+  <div class="container-fluid">
+    <!-- Brand and toggle get grouped for better mobile display -->
+    <a class="nav-link link-primary mx-2 " style="text-decoration: none !important;" href="{{ url_for('cases.case', institute_id=institute._id, case_name=case.display_name) }}" rel="noopener" target="_blank">Case {{case.display_name}}</a>
 
-  <!-- Collect the nav links, forms, and other content for toggling -->
-  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarText" aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
-      <span class="navbar-toggler-icon"></span>
-  </button>
+    <!-- Collect the nav links, forms, and other content for toggling -->
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarText" aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
Not your doing, but this isn't really active right now anyway, is it? We may as well drop it. 😊

Reply:
I think it's the button used to expand the menu when having a small screen/window?

![image](https://github.com/Clinical-Genomics/scout/assets/112548186/3686117c-6fce-4d03-ae8a-6ab4a3b8101c)

Reply:
It's the button to expand the menu if using a small screen/window I believe?

![image](https://github.com/Clinical-Genomics/scout/assets/112548186/f7006fff-568b-401c-a029-2413e059de08)

Reply:
![image](https://github.com/Clinical-Genomics/scout/assets/112548186/7961dd61-b79d-4157-bed6-56e7484f0704)